app/controllers/cart_controller.rb

```diff
@@ -11,7 +10,27 @@ def update
     session[:cart][item_id_str] ||= 0
     session[:cart][item_id_str] = session[:cart][item_id_str] + 1
```

Comment:
Just wanna make sure I understand the difference here. With the update method, we add an item using `session[:cart][item_id_str] + 1`. But with the add_subtract method, we add an item using `@contents[item_id] += 1` in the cart model. Is the effect the same?